packages/execution-environments/tsconfig.build.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "typeRoots": ["../../node_modules/@types", "./node_modules/@types"]
+  },
+  "include": ["./src", "./src/openrpc.json"],
+  "exclude": [
+    "**/*.test.ts",
+    "./src/**/test-utils",
```

Comment:
Does this remove them from the dist as well? We have an open issue for that: https://github.com/MetaMask/snaps-monorepo/issues/843

Reply:
It does!